group-coordinator/src/main/java/org/apache/kafka/coordinator/group/common/TopicIdPartition.java

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
We already have the very same class in the `metadata` module. I wonder if we should move that one to `server-common` module so that we could reuse it here. It could land in the `common` package over there. What do you think?

Reply:
Its not the exact same, that one has topicId mapped to topicPartition here its just the partition number

Reply:
I was referring to https://github.com/apache/kafka/blob/trunk/metadata/src/main/java/org/apache/kafka/controller/TopicIdPartition.java. It is really bad to have two classes with the same name but slightly different. I wonder if we can do something about it...

Reply:
Ohh my bad I got this question before so I instinctively replied, I didn't know this class existed, yeah this seems usable, would we need a new PR to move the file though?

Reply:
Yeah, we can have a separate PR to move that class (and add some javadoc to it). I am actually annoyed by the fact that we have two TopicIdPartition classes but with different content. One with (id, name, partition) and one with (id, partition). This is really misleading...

Reply:
yeah that's true I agree, I wasn't sure what else to call this one, okay I will add a new PR for this.